src/components/CategoryList.jsx

```diff
@@ -13,8 +13,8 @@ const CategoryList = ({ className, limit = Number.MAX_VALUE }) =>
         label={category.label}
         icon={category.iconName}
         iconStyle={{
-          color: category.color,
-          backgroundColor: category.backgroundColor,
+          color: category.color, // RGB icon color
+          backgroundColor: category.color + '28', // RGBA background color equal to the icon color + 0.157 alpha
```

Comment:
Great if this is the targetting color is the same ! I tried to do something similar to this once, without success

Reply:
Nice and simple way to do that indeed :+1:

Now **let's move it to an util function**, so it's easier to name/comment/debug/test/make evolve. And re-use, as I see you use the same operation in this PR.